virtualizarr/writers/kerchunk.py

```diff
@@ -67,13 +90,65 @@ def remove_file_uri_prefix(path: str):
         return path
 
 
+def convert_v3_to_v2_metadata(
```

Comment:
Should this (a) not live in the kerchunk-specific writer module, (b) actually live in zarr-python upstream? Or is there no use for it upstream?

Reply:
>(a) not live in the kerchunk-specific writer module

This is the only module that requires it, but I'm open to moving it, should I put it in `utils.py`?

>(b) actually live in zarr-python upstream? Or is there no use for it upstream?

I'm not sure if there is a use for it, but I could create another issue if you think it is warranted to ask.

Reply:
Let's just put it in `utils.py` for now, as it's not specifically related to kerchunk.

@d-v-b is there anything like this / any need for this function upstream?